jet/kafka-connect/src/main/java/KafkaConnectSample.java

```diff
@@ -0,0 +1,88 @@
+ * </pre>
+ * <p>
+ * To generate a new items externally go to <a href="http://localhost:7474/browser/">Neo4j Browser</a>
+ * (no authentication required) and execute the following query
+ * <pre>
+ *     CREATE (:TestSource {name: 'my-name', value: 'my-value', timestamp: datetime().epochMillis})
+ * </pre>
+ */
+public class KafkaConnectSample {
+
+    //You can download the newer version from https://www.confluent.io/hub/neo4j/kafka-connect-neo4j/
+    private static final String CONNECTOR_URL = "https://repository.hazelcast.com/download"
+            + "/tests/neo4j-kafka-connect-neo4j-2.0.1.zip";
+
+    private static final String BOLT_URL = "bolt://localhost:7687";
+
+    public static void main(String[] args) throws Exception {
+        Properties connectorProperties = new Properties();
+        connectorProperties.setProperty("name", "neo4j");
+        connectorProperties.setProperty("connector.class", "streams.kafka.connect.source.Neo4jSourceConnector");
+        connectorProperties.setProperty("topic", "some-topic");
+        connectorProperties.setProperty("neo4j.server.uri", BOLT_URL);
+        connectorProperties.setProperty("neo4j.streaming.poll.interval.msecs", "1000");
+        connectorProperties.setProperty("neo4j.streaming.property", "timestamp");
+        connectorProperties.setProperty("neo4j.streaming.from", "ALL");
+        connectorProperties.setProperty("neo4j.source.query",
+                "MATCH (ts:TestSource) WHERE ts.timestamp > $lastCheck RETURN ts.name AS name, ts.value AS value, ts.timestamp AS timestamp");
+
+        insertNodes("items-1");
+
+        Pipeline pipeline = Pipeline.create();
+        StreamStage<SourceRecord> streamStage = pipeline.readFrom(KafkaConnectSources.connect(connectorProperties))
+                .withoutTimestamps()
+                .setLocalParallelism(2);
+        streamStage.writeTo(Sinks.logger());
+
+        JobConfig jobConfig = new JobConfig();
+        jobConfig.addJarsInZip(new URL(CONNECTOR_URL));
+
+        Config hzConfig = new Config();
+        hzConfig.getJetConfig().setEnabled(true);
+        hzConfig.getJetConfig().setResourceUploadEnabled(true);
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance(hzConfig);
+        JetService jet = hz.getJet();
+        System.out.println("Downloading the connector jar and submitting the job...");
+        Job job = jet.newJob(pipeline, jobConfig);
+        System.out.println("Job submitted");
+
+        insertNodes("items-2");
+
+        job.join();
+    }
+
+    private static void insertNodes(String prefix) {
+        try (Driver driver = GraphDatabase.driver(BOLT_URL, AuthTokens.none()); Session session = driver.session()) {
+            for (int i = 0; i < 100; i++) {
+                session.run("CREATE (:TestSource {name: '" + prefix + "-name-" + i + "', value: '"
+                        + prefix + "-value-" + i + "', timestamp: datetime().epochMillis});");
```

Comment:
To make it a bit more real world let's make this 

```
CREATE (:Person {firstName:'..', lastName:'..', role:'..' 
```

and in the pipeline map it to a bean before printing to logger. Now it prints bunch of Kafka connect json and is not very nice.

Reply:
Done https://github.com/hazelcast/hazelcast-code-samples/pull/569/commits/85d301702c2c87c19d5dd626c19f62f7445d68be